graphQl/resolvers.js

```diff
@@ -1,31 +1,61 @@
-const bcrypt = require('bcrypt');
-const validator = require('validator');
-// const jwt = require('jsonwebtoken');
-const User = require('../models/User');
-
+const bcrypt = require("bcrypt");
```

Comment:
We have configured ESLint to use single quotes instead of double, please check if you have set up ESLint locally properly.